flip.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency("activesupport", ">= 4.0", "<= 5.2")
```

Comment:
activesupport uses good deprecation and consistent migration paths for end users. Gems should typically avoid upper bounding on widely shared and well tested dependencies, unless there are known incompatibilities.